pkg/config/config.go

```diff
@@ -68,6 +69,32 @@ func LoadConfig() *Config {
 		return &Config{}
 	}
 
+	h, err := hasOldConfig()
+	if err != nil {
+		fmt.Printf("Unable to determine if old config exists: %s", err)
+	}
+
+	if h {
+		userHomeDir := getUserHomeDir()
+		oldLocation := path.Join(userHomeDir, configDirHidden)
+		fmt.Printf("Attempting to move config from: %s/%s to %s\n", oldLocation, configFile, fullConfigFile)
```

Comment:
I would prefer if this only print a maximum of one message (either one saying that the file was moved successfully or one saying that there was an error).

There's also the problem that this runs before the server starts listening which means that this messages being print before the "API server listening" message, which will break frontends.

Reply:
@aarzilli I consolidated the messages into one and removed a few other extra empty lines to better match the style that was already present in the file.

Can you elaborate on the API server listening message problem?
When I test it locally I get the message after the "API server listening at:" message and the current file also prints messages out in the same way that I am doing now.

Reply:
The problem is that clients will look for the "API server listening at:" and expect it to be the first thing on stdout. I think those messages should go to stderr `fmt.Fprintf(os.Stderr, ...` so they don't interfere with that.